packages/module/src/DetailsPageHeader/utils/ActionButtons.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button, Flex, FlexItem, Tooltip } from '@patternfly/react-core';
+import React from 'react';
+
+export interface ActionButtonProps {
+  /** Text rendered inside the button */
+  label: string;
+  /** A callback for when the action button is clicked */
+  callback: (event: React.MouseEvent) => void;
+  /** Adds button variant styles */
+  variant?: 'primary' | 'secondary' | 'tertiary' | 'link';
+  /** ID for the action button */
+  id?: string;
+  /** Adds disabled styling and disables the button using the disabled html attribute */
+  isDisabled?: boolean;
+  /** Text for a tooltip */
+  tooltip?: string;
+};
+
+export interface ActionButtonsProps {
+  /** Array of action buttons */
+  actionButtons: ActionButtonProps[];
+};
+
+export const ActionButton: React.FunctionComponent<ActionButtonProps> = ({
+  label,
+  callback,
```

Comment:
Can we keep the interface to match the original Button?

Reply:
Lets match the props 👍

Reply:
Updated.